.github/workflows/unit.yaml

```diff
@@ -18,11 +18,11 @@ jobs:
       - uses: pre-commit/action@v3.0.0
 
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
I would prefer to remain on ubuntu-latest and drop older version of Python. 3.7 has reached EOL a long time ago I think we can safely drop it.

Reply:
Updated. I considered that initially, but didn't think an unsolicited third party PR to drop python 3.7 support would be reasonable